### PR TITLE
4.2 update

### PR DESCRIPTION
**Pull request by Ermad**
Update for 4.2, along with general update for crafting equations.

Also has a few various bug fixes.

### CODE REVIEW

File: app/js/ffxivcraftmodel.js [hunk 1]
```diff
@@ -73,67 +73,58 @@ Synth.prototype.calculateBaseProgressIncrease = function (levelDifference, craft
...
-    if (crafterLevel >= 260) {
-        baseProgress = 1.834712812e-5 * craftsmanship * craftsmanship + 1.904074773e-1 * craftsmanship + 1.544103837;
-    }
-    else if (crafterLevel >= 120) {
-        baseProgress = 2.09860e-5 * craftsmanship * craftsmanship + 0.196184 * craftsmanship + 2.68452;
-    }
-    else if (crafterLevel < 120) {
-        baseProgress = 0.214959 * craftsmanship + 1.6;
-    }
+    baseProgress = 1.834712812e-5 * craftsmanship * craftsmanship + 1.904074773e-1 * craftsmanship + 1.544103837;
```
Comment (doxxx) [inline app/js/ffxivcraftmodel.js]: Have you been able to test this with lower level crafters?

Comment (Ermad) [reply app/js/ffxivcraftmodel.js]: Yea, there was a bunch of 1-50 data in various issues on here I was able to test against. It was within a point for all the ones I could find.

**Comment by doxxx** [comment]
Thanks!

**Comment by doxxx** [comment]
Changes deployed to beta site.

**Pull request merged**